ArduCopter/motors.cpp

```diff
@@ -181,7 +182,21 @@ void Copter::motors_output()
     }
 
     // push all channels
-    SRV_Channels::push();
+    if (full_push) {
+        // motor output including servos and other updates that need to run at the main loop rate
+        SRV_Channels::push();
```

Comment:
do we end up with an unpaired cork/push?

Reply:
No, all checks out. I have added in the check however.